### ISSUE TITLE
UI: Changing (or adding) cover photo needs response to the user

### ISSUE THREAD

**Issue by tschlotfeldt** [redmatrix/hubzilla#1076]
When I first figured out the feature of changing a cover photo I didn't recognized what happened. There was no positive (or negative) response, that the change happend. In my case I added a cover for five times till I got it the change always happened.

Possible solution: After adding or changing the cover photo give the user a small feedback like: „The cover photo [filename] has been added to your channel [klickable channel name].“

**Comment by git-marijus** [comment]
I could not reproduce this issue. After submission of the new cover photo i am being redirected to the channel page where i see the new cover photo.
Can you post the steps to reproduce your issue?

**Comment by tschlotfeldt** [comment]
Yes, I was also redirected to my channel page. But I didn't understand that that it is. I also didn't know exactly what this cover photo is and where will it be displayed. I think what also confused me is the fact, that the channel cover photo just slides away and you can't get it back by scrolling up.

I think it would be helpful if you give the user a more specific feedback: Everythink is alright, the cover photo has been added and when you klick here you can see it.

**Comment by git-marijus** [comment]
> you can't get it back by scrolling up

You can, you just have to try harder :smile:.  There is a scroll resistance built in so that you don't always scroll into the cover if you scroll up on your channel page.

> I think it would be helpful if you give the user a more specific feedback: Everythink is alright, the cover photo has been added and when you klick here you can see it.

That is what i mean. There is no need to click anywhere after submission to see it. You already look at it.

We might need a short description on the upload page to make it clear what a cover photo is and where it is visible. This can be different in a different theme though...
